synapse/cortex.py

```diff
@@ -854,6 +854,21 @@ class Cortex(s_oauth.OAuthMixin, s_cell.Cell):  # type: ignore
             'description': 'Enable Storm scrape interfaces when using $lib.scrape APIs.',
             'type': 'boolean',
         },
+        'storm:query:mirror': {
+            'description': 'Configure a Cortex mirror or AHA pool to offload Storm queries to. This is an experimental option and should not be used in production.',
+            'type': 'object',
+            'properties': {
+                'url': {
+                    'type': 'string',
+                    'description': 'Telepath URL for a Cortex mirror or AHA pool.'
+                },
+                'timeout': {
+                    'type': 'integer',
```

Comment:
What is the meaning of a negative timeout value [which is allowed by schema...] ?

Reply:
Same thing as a <= 0 timeout to `wait_for`, successful if already caught up, otherwise timeout immediately.